docs/knowledgebase/integrate/memory-profiling.md

```diff
@@ -0,0 +1,75 @@
+# Memory profiling utilizing Substrate
+
+This implementation uses the linux system profile, utilizing memory allocations of Substrate & Polkadot.
```

Comment:
Maybe add some info about why you'd want to do memory profiling? How memory usage influences operation? One thing to check with @gnunicorn , but would runtime features/bugs cause memory issues, or is this a client thing that runtime developers wouldn't need to worry about?